Documentation/Compass/README.md

```diff
@@ -35,27 +35,26 @@ Compass:
     /// - Parameters:
     ///   - viewpointRotation: The viewpoint rotation whose value determines the
     ///   heading of the compass.
-    ///   - autoHide: A Boolean value that determines whether the compass
-    ///   automatically hides itself when the viewpoint rotation is 0 degrees.
-    public init(viewpointRotation: Binding<Double>, autoHide: Bool = true)
+    ///   - action: An action to perform when the compass is tapped.
+    public init(viewpointRotation: Binding<Double>, action: (() -> Void)? = nil)
 ```
 
 ```swift
     /// Creates a compass with a binding to an optional viewpoint.
     /// - Parameters:
     ///   - viewpoint: The viewpoint whose rotation determines the heading of the compass.
-    ///   - autoHide: A Boolean value that determines whether the compass automatically hides itself
-    ///   when the viewpoint's rotation is 0 degrees.
-    public init(viewpoint: Binding<Viewpoint?>, autoHide: Bool = true)
+    ///   - action: An action to perform when the compass is tapped.
+    public init(viewpoint: Binding<Viewpoint?>, action: (() -> Void)? = nil)
 ```
 
-`Compass` has the following modifier:
+`Compass` has the following modifiers:
 
 - `func compassSize(size: CGFloat)` - The size of the `Compass`, specifying both the width and height of the compass.
+- `func automaticallyHides(_:)` - Specifies whether the ``Compass`` should automatically hide when the heading is 0.
 
 ## Behavior:
 
-Whenever the map is not orientated North (non-zero bearing) the compass appears. When reset to north, it disappears. An initializer argument allows you to disable the auto-hide feature so that it always appears.
+Whenever the map is not orientated North (non-zero bearing) the compass appears. When reset to north, it disappears. The `automaticallyHides` view modifier allows you to disable the auto-hide feature so that it is always displayed.
```

Comment:
Should this then be named more like how SwiftUI names modifiers like this? 

maybe `disableAutoHide(:)`?

<img width="381" alt="image" src="https://user-images.githubusercontent.com/3718875/229574369-2431a8af-ae68-46c8-84e7-2f739136b139.png">

Reply:
That would be OK.  @dfeinzimer?

Reply:
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/291

`disableAutocorrection` was deprecated in favor of `autocorrectionDisabled` so I'll follow the new scheme. e.g. `autoHideDisabled`.